spacy/tests/test_requirements.py

```diff
@@ -0,0 +1,77 @@
+import re
+from pathlib import Path
+
+
+def test_build_dependencies(en_vocab):
+    # Check that library requirements are pinned exactly the same across different setup files.
+    libs_ignore_requirements = ["pytest", "pytest-timeout", "mock", "flake8", "jsonschema"]
+    libs_ignore_setup = ["fugashi", "natto-py", "pythainlp"]
+
+    # check requirements.txt
+    req_dict = {}
+    try:
+        # for CLI usage
+        root_dir = Path(__file__).parent.parent
+        req_file = root_dir / "requirements.txt"
+        with req_file.open() as f:
+            lines = f.readlines()
+    except FileNotFoundError as e:
+        try:
+            # for local usage
+            root_dir = Path(__file__).parent.parent.parent
+            req_file = root_dir / "requirements.txt"
+            with req_file.open() as f:
+                lines = f.readlines()
+        except FileNotFoundError as e:
+            # where areth thou ?
```

Comment:
😂

If it turns out we need these conditions, we could probably simplify this part and avoid the `try`/`excepts`:

```python
roots = [Path(__file__).parent.parent, Path(__file__).parent.parent.parent]  # or whatever
for root_dir in roots:
    req_file = root_dir / "requirements.txt"
    if req_file.exists():
        # read file etc.
        break
```